SpaceDash/Info.plist

```diff
@@ -11,7 +11,7 @@
 	<key>CFBundleIcons~ipad</key>
 	<dict/>
 	<key>CFBundleIdentifier</key>
-	<string>com.pushpinderpalsingh.SpaceDash</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
```

Comment:
Why have you changed this?

Reply:
Without changing this project will not build in my computer!

Reply:
Bundle identifier should be unique!